notes.tex

```diff
@@ -100,14 +103,53 @@ \section{Sections}\label{sec:sections}
   is an arrow $s:X\to Y$ such that $p\circ s=1_X$ in $\cat{C}$.
 \end{definition}
 
+\begin{remark}
+The existence of a section implies that the map $p$ is surjective. This is obvious from the definition.
+The converse depends on the axiom of choice. If $p$ is surjective, then there exists a section of $p$ if we accept the axiom.
+\end{remark}
+
 \begin{definition}
   Let $X,Y$ be a topological space, let $U\subseteq X$ be open, and let $p:Y\to X$ be
   a continuous map. Then the \emph{sheaf of sections of $p$ at $U$} is the set
   $\Gamma\rr{U,p}$ of all continuous maps $s:U\to X$ such that, for all $x\in U$,
   $\rr{p\circ s}\rr{x}=x$.
 \end{definition}
 
+\begin{example}
+Let let $p : \mathbb{R} \to \mathbb{R}$ be the continuous map $x \mapsto x^2$. We have that
+\begin{align*}
+	\Gamma\rr{U,p} = \{\text{set of square root functions on } U\}.
+\end{align*}
+For example, if $U = ]1, 2[$ then
+\begin{align*}
+	\Gamma\rr{U,p} = \{+\sqrt{\phantom{a}}, -\sqrt{\phantom{a}}\}.
+\end{align*}
+In particular, a square root function can be described as a continuous map $\sigma  : U \to \mathbb{R}$ such that for every  $u \in U$ we have $\sigma(u)^2 = u$.
+\end{example}
+
+\begin{example}
+\label{ex:complexroot}
+Consider the continuous map $p : \mathbb{C}^\times \to \mathbb{C}^\times$ sending $z \mapsto z^2$. Note that this map is not continuous if considered over all of $\mathbb{C}$. If we "confine ourselves to a quadrant on the circle" (i.e. open sets $U$ contained in one quadrant) we have that
+\begin{align*}
+	\# \Gamma\rr{U, p} = 2.
+\end{align*}
+This makes sense, as the inverse function $z^{1/2}$ is a multivalued function with exactly $2$ values. What we mean by the "confinement to the quadrant" is that if we happened to have a union of open sets that somehow "cross" the removed point $0$, then the sheaf of sections will have no sections at all. This makes sense insofar as such a section would not be continuous near $0$. In particular, the sheaf of scetion seems to know how the open sets are distributed.
+\end{example}
+\note{This example is a bit poorly explained, but I think is good to consider for intuition, let's discuss how to fix this? Come back to this example and how it becomes problematic when gluing - disagreement on overlap}
+
 \missingexample
+\note{add more complicated example}
+
+
+For two topological spaces $X$ and $Y$ and a continuous map $p : X \to Y$, sheaves of sections give us information about how homeomorphic the map $p$ is.
+
+\begin{proposition}
+Let $p : X \to Y$ be a continuous map. If $p$ is a homeomorphism then $\Gamma\rr{U,p}$ consists of the continuous map $g : Y \to X$ such that $g \circ f = 1_X$ and $f \circ g = 1_Y$. 
+\end{proposition}
+
+\begin{remark}
+We will note this formally later, however something to think about is that if a map $p : X \to Y$ is locally homeomorphic then we can recover $p$ from its sheaf of sections. In particular, we will see that this relates to the concept of "gluing" things together.
```

Comment:
I have an irrational hatred for "putting things in quotes". In this case, I think gluing is a technical term that we can use freely. Maybe make it more formal by removing the 'things together'.